### PR TITLE
Merge in City damage report 202408

### PR DESCRIPTION
**Pull request by tallcoleman**
Script changes:

- Converted to standalone script instead of jupyter notebook and re-organized file structure
- Added new functionality to the script to only show damage reports within a certain date range. See the new README for details.

Report features:

- Includes all relevant damage reports from 2023 onwards (20 in total, 13 of them are net new compared to the previous report). All of the 20 appear to be related to post and rings, no racks.
- Report includes photos and confirmed asset IDs (to link to City open data) from surveys
 
Report changes:

- The MatchesDisplay tab now includes survey photos as well as the thumbnail map indicating the location. Photos are pulled from the `/photos` directory based on name and resized before insertion.
- New field “City_Problem_Type” indicates the main issue reported, e.g. “Loose/unsecured”, “Post with no ring”
- New fields “Check_311_Date” and “Check_311_Notes” to record whether there are 311 reports for the issue already (though the utility of this is limited, since the City 311 page only shows 311 reports from the last 30 days at most)
- New MatchesTable tab shows the report with the first (nearest or confirmed) linked data point from the City

### CODE REVIEW

File: .gitignore
Comment (sonalranjit) [inline .gitignore:1]: you probably want to ignore this change until the `develop` branch is merged into `main` to avoid conflicts

**Review by sonalranjit — CHANGES_REQUESTED** [review]
Thanks @tallcoleman , awesome to have everything in one place. I've just mentioned the changes for the `.gitignore` other than that we should be good to merge

File: .gitignore
Comment (tallcoleman) [reply .gitignore:1]: Fixed by #146

**Comment by tallcoleman** [comment]
Did some additional clean up of this script:

- Added [uv](https://docs.astral.sh/uv/) as an python/dependencies manager
- Separated out tests folder
- Updated date parsing to match current api format (see: #184 )
- Updated dashboard link generation to point to new dashboard

**Review by sonalranjit — APPROVED** [review]
Coool! thanks @tallcoleman

**Pull request merged**